res/WELCOME.md

```diff
@@ -17,7 +17,7 @@ It is in general a good idea to await the automated test ("Checks") in this Pull
 
 Before you can contribute the patches, your GitHub username needs to be added to the list of permitted users. Any already-permitted user can do that, by adding a PR comment of the form `/allow <username>`. A good place to find other contributors is the channel [`#git-devel`](https://webchat.freenode.net/#git-devel) on the FreeNode IRC network.
 
-    <newcontributor> I've just created my first PR, could someone /allow me? https://github.com/gitgitgadget/git/pull/12345
+    <newcontributor> I've just created my first PR, could someone please /allow me? https://github.com/gitgitgadget/git/pull/12345
```

Comment:
How about squashing that patch that adds the word "please" into the previous commit?

In the Git project, we prefer patch series without any "oops, I actually meant to do this, too" patches, i.e. we recommend to rewrite and polish the patches e.g. via interactive rebase until they shine, so to say.